ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/validation/CustomValidators.java

```diff
@@ -99,7 +100,8 @@
   SHOW_COLUMNS(ShowColumns.class, ListSourceExecutor::columns),
   EXPLAIN(Explain.class, ExplainExecutor::execute),
   DESCRIBE_FUNCTION(DescribeFunction.class, DescribeFunctionExecutor::execute),
-  DESCRIBE_CONNECTOR(DescribeConnector.class, new DescribeConnectorExecutor()::execute),
+  DESCRIBE_CONNECTOR(DescribeConnector.class,
+      new DescribeConnectorExecutor(new DefaultConnectServerErrors())::execute),
```

Comment:
Unrelated to your PR but what is this validation actually doing? AFAICT `DescribeConnectorExecutor::execute()` never actually throws an error. Does this validation do anything or is it a no-op?

Reply:
I agree with your judgement that `DescribeConnectorExecutor::execute()` never throws, so it's a no-op. 

After a brief look at this class I think that only very limited number of executors ever throw, consequently only these validators actually do any useful work, namely `InsertValuesExecutor()::execute`, `ListSourceExecutor::columns`, and `ExplainExecutor::execute`. 

Maybe there is a reason for this, but it appears to me that executor behavior is inconsistent - either all executors should throw on errors or return an error entity instead. WDYT? If you agree, I'll open an issue.